easybuild/easyconfigs/u/util-linux/util-linux-2.39-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'util-linux'
+version = '2.39'
+
+homepage = 'https://www.kernel.org/pub/linux/utils/util-linux'
+
+description = "Set of Linux utilities"
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['%s/v%%(version_major_minor)s' % homepage]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['b1f597ad172a2ee17b0a7ae4be7ab7e1b1a6f9394ddbd3d8ec90ae4ed7333579']
```

Comment:
Please add https://github.com/util-linux/util-linux/commit/ffdd75cceedafac3ba7a80b86e1b103c383b260e to fix the [build failure](https://github.com/easybuilders/easybuild-easyconfigs/pull/18023#issuecomment-1575572091)